src/components/Nav/Nav.Props.ts

```diff
@@ -21,6 +21,11 @@ export interface INavProps {
    * Indicates whether the navigation component renders on top of other content in the UI
    */
   isOnTop?: boolean;
+
+  /**
+   * (Optional) The key of the nav item initially selected.
+   */
+  initialSelectedKey?: string;
```

Comment:
What if you called this defaultSelectedKey to be consistent with other uncontrolled properties? (e.g. defaultChecked)